hummingbot/strategy/market_trading_pair_tuple.py

```diff
@@ -20,3 +26,18 @@ def quote_balance(self):
     @property
     def base_balance(self):
         return self.market.get_balance(self.base_asset)
+
+    def get_price(self, is_buy: bool) -> Decimal:
+        return Decimal(self.order_book.get_price(is_buy))
```

Comment:
This is not ok... there's no quantization and so the decimal object returned would have many digits.

You should use `self.market.get_price()` instead, since that one is quantized.